admin-notice/js/script.js

```diff
@@ -29,10 +29,9 @@ function onDismissed(event) {
 }
 
 function registerDismissHooks() {
-    const notices = document.getElementsByClassName("vip-notice");
-    for (const notice of notices) {
-        // Hooking up on the event caused by the core implementation of is-dismissible class, which generates dismiss button.
-        notice.addEventListener("DOMNodeRemoved", onDismissed);
+    const buttons = document.getElementsByClassName("notice-dismiss");
+    for (const button of buttons) {
+        button.onclick = onDismissed;
```

Comment:
Is this going to overwrite other events that are registered on these elements? Can we just use jQuery?

Reply:
It will. Would have the use addEventListener to append click events without removing the rest at the minimum.

Should use jQuery though since that way the compatibility exactly matches WordPress so we don't end up with any     surprises like our dismiss notice not working while all other JS does or the opposite.

Reply:
From my earlier conversation with @GaryJones use of JQuery is discourage.

Good point with overwriting non-vip-admin notices buttons. Maybe we should add back a `vip-admin-notice` on the div, then get all those diffs. And then get all the buttons in those gifts to avoid rewriting some other buttons.

Reply:
👍 I implemented this in 1e284b42e